### ISSUE TITLE
Signature OTP

### ISSUE THREAD

**Issue by Yorxxx** [Yorxxx/ok-fake-server#12]
Should implement the method that validates the signature

## Request
------

    URL http://localhost:8000/api/transactions/{transaction-id}/signature_otp/
    Request Method:POST
    Status Code:404 Not Found
    Remote Address:127.0.0.1:8000
    Referrer Policy:no-referrer-when-downgrade

## Request Headers
------
    Accept:application/json, text/plain, /
    Accept-Encoding:gzip, deflate, sdch, br
    Accept-Language:es-ES,es;q=0.8
    authorization:Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjkyNiwiaXNzIjoiaHR0cDpcL1wvbG9jYWxob3N0OjgwMDBcL2FwaVwvYXV0aGVudGljYXRlIiwiaWF0IjoxNDkxOTkxMDE0LCJleHAiOjE0OTE5OTQ2MTQsIm5iZiI6MTQ5MTk5MTAxNCwianRpIjoiYzMzZDRkMWRkZThhMDRiNWJjNWU5NTJjMWMwOTY5ZTUifQ.DtqxJm-2WesBkeT8LX3ynJw_Vfu_ZFidUroGoQw5Yek

## Request Payload

```javascript
{"signature_positions":["1", "2", "5"], "signatureData":["1", "2", "4"]}
```

**Comment by Yorxxx** [comment]
According to p2p source

```javascript
...
ticket = response.data.ticket;
if(response.status === 200) {
   toast.loading.hide();
   $scope.openPopUp(vm.currentTransactionId);
}
```
Expected response should contain a ticket object and 200 status code.

**Comment by Yorxxx** [comment]
Implemented on #13
